components/library/webkitgtk/patches/06-wasm-section-parser.patch

```diff
@@ -0,0 +1,38 @@
+--- a/Source/JavaScriptCore/wasm/WasmSectionParser.cpp	2023-08-09 09:49:33.123924000 +0000
```

Comment:
Could you please add a comment explaining why is this patch needed?  Is it an upstream bug, or OpenIndiana bug?  Was the bug reported already?

Thank you.